crates/sui-config/tests/snapshots/snapshot_tests__empty_genesis_snapshot_matches.snap.new

```diff
@@ -0,0 +1,7 @@
CwELAEAgAAAAAAAAAABAFgAAAAAAAAAAQCwAAAAAAAAAABIADAQOBBEHDQQPBBULBAIOAwAAAQQLABAEQSwCDwAAADoXCgERFwoBLjgHIAMIBRIKAUUWDAIKAAsCOAgMAwsDES4FAgsBAQsAAQIQAAAAIBIKAS44CSADBgUNCgFFIAwCCgALAkQgBQALAAELAQECEQMAABEPCgAPAAsBEQoMAwsDCwIRLwoALhEHCwAPBBUCEgMAAD4VCgMuETAMBQoADwALBREKDAQLBAsBCwILAxEoCgAuEQcLAA8EFQITAwAACSEKABAADgERBSADBwUOCgAQAg4BEQUgDAIFEAkMAgsCAxYLAAEGAAAAAAAAAAAnCgAPAgsBRCAKAC4RBwsADwQVAhQDAABAHgoAEAALAREJDAUOBTgFAwwLAAEGAAAAAAAAAAAnDQU4BgwECgAPAAsEQyAMAwsDCwIRMQoALhEHCwAPBBUCFQMAAEEnCwERMAwCCgAQAAsCEQkMBA4EOAUDDwsAAQYAAAAAAAAAACcNBDgGDAMKABADDgM4CiADHAsAAQYAAAAAAAAAACcKAA8DCwNEFgoALhEHCwAPBBUCFgMAAD4WCgQuETAMBgoADwALBhEKDAULBQsBCwILAwsEETMKAC4RBwsADwQVAhcAAABEPAoALkEWDAgGAQAAAAAAAAAMBgoGCggjAwsFOQoACgYMAgwBCwEuCwJCFhQMBQoGDAcKBwYAAAAAAAAAACQDHAU0CwcGAQAAAAAAAAAXDAcKAAoHDAQMAwsDLgsEQhYUCgUkAy0FNAoACgcKBwYBAAAAAAAAABZHFgUXCwYGAQAAAAAAAAAWDAYFBgsAAQIYAQAAAQQLABAFFAIZAQAAAQQLABABFAIaAQAAEggLABAACwERCwwCCwIRIAIbAQAAEggLABAACwERCwwCCwIRIgIcAQAAEggLABAACwERCwwCCwIRHwIAAwAAAAQABQAGAAEAAgAAAA9lcG9jaF90aW1lX2xvY2uXAqEc6wsFAAAACgEABAIECAMMFAUgGQc5RQh+FAaSARQKpgEFDKsBPg3pAQIAAAABAAIEAAEDAgAABAABAAAFAgMAAAYEBQABBQYDAAIIAAcIAQABBggAAQMCAwcIAQEIAAEGCAEPZXBvY2hfdGltZV9sb2NrCnR4X2NvbnRleHQNRXBvY2hUaW1lTG9jawlUeENvbnRleHQHZGVzdHJveQVlcG9jaANuZXcAAAAAAAAAAAAAAAAAAAAAAAAAAgMIAAAAAAAAAAADCAEAAAAAAAAAAAIBBQMAAQAAAwwLABMADAILAS4RAwsCJgMLBwEnAgEBAAABBAsAEAAUAgIBAAABCwsBLhEDCgAjAwgHACcLABIAAgAAAA9lcmM3MjFfbWV0YWRhdGGrA6Ec6wsFAAAACQEACAIIFAMcKAVELAdwjQEI/QEoCqUCEQy2AkUN+wIGAAABAQACAAMABAQAAAUFAAMGBwACCwcAAQYHAAAHAAEAAAgCAwAACQQFAAAKAAYAAAwABwABDgoJAAMPCgsAAhAJDAABBggAAQYIAgMIAQoCCgIBCAABAwEIAQEGCAEBBggDAAEIBAEKAgEIAgEIAw9lcmM3MjFfbWV0YWRhdGEFYXNjaWkDdXJsBHV0ZjgORVJDNzIxTWV0YWRhdGEHVG9rZW5JRAZTdHJpbmcEbmFtZQNuZXcMbmV3X3Rva2VuX2lkCHRva2VuX2lkA1VybAl0b2tlbl91cmkCaWQGc3RyaW5nDXN0cmluZ191bnNhZmUKbmV3X3Vuc2FmZQAAAAAAAAAAAAAAAAAAAAAAAAACAAAAAAAAAAAAAAAAAAAAAAAAAAEAAgMKCAEHCAIMCAMBAgENAwABAAAIAwsAEAACAQEAAAkKCwIRBQwDCwALAREGCwMRBxIAAgIBAAAIAwsAEgECAwEAAAgDCwAQAQIEAQAACAMLABACAgABAAAAAgATZXBvY2hfcmV3YXJkX3JlY29yZOgDoRzrCwUAAAALAQAIAggMAxQfBDMCBTUkB1nSAQirAhQKvwIQDM8CWg2pAwoPswMEAAIAAwAEAAUABggAAwgCAAENBAAABwABAAAJAgMAAAoEAQAACwQFAAERBgcAAhIJAwEIBQgCBwgAAwEDBgMDAwMFBwgBAAEGCAABBQEHCAEBCAIBCAABCQAKc3VpX3N5c3RlbQ12YWxpZGF0b3Jfc2V0E2Vwb2NoX3Jld2FyZF9yZWNvcmQGb2JqZWN0CHRyYW5zZmVyCnR4X2NvbnRleHQRRXBvY2hSZXdhcmRSZWNvcmQMY2xhaW1fcmV3YXJkCVR4Q29udGV4dAZjcmVhdGUFZXBvY2gJdmFsaWRhdG9yBGluZm8ESW5mbxJjb21wdXRhdGlvbl9jaGFyZ2ULdG90YWxfc3Rha2UPZGVsZWdhdG9yX2NvdW50A25ldwxzaGFyZV9vYmplY3QAAAAAAAAAAAAAAAAAAAAAAAAAAgACBgwIAgoDDgMPAxADCwUAAwAAAxIKABAAFAYBAAAAAAAAABcKAA8AFQsBCgAQARQYCwAQAhQaAgEDAAADCgsFEQQLAAsBCwILAwsEEgA4AAICAQAAAwQLABADFAIDAQAAAwQLABAEFAIABAACAAMAAQAFAAAAAQADIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIKc3VpX3N5c3RlbQ5TdWlTeXN0ZW1TdGF0ZQAAcAAAAAAAAAAAAAAAAAAAAAAAAAAFAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQHoQ81oAAABAehDzWgAAAEB6EPNaAABkAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAACIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
this is expected: https://github.com/MystenLabs/sui/blob/036462af58a6acf6ec88465bf386b1164d70dccd/crates/sui-config/src/genesis.rs#L244